### DISCUSSION TITLE
No smtp found

### DISCUSSION THREAD

**Discussion by Kevin3-00** [DrPython3/MailRipV2#13] [General]
First I commend your efforts in making this tool,I loaded combos of 1.5 million lines sometimes lesser but at the end no single smtp found.What could be the reason since I checked all the settings and followed your instructions.Please I will like to know if this tool is working.

**Comment by toms199668** [comment]
> First I commend your efforts in making this tool,I loaded combos of 1.5 million lines sometimes lesser but at the end no single smtp found.What could be the reason since I checked all the settings and followed your instructions.Please I will like to know if this tool is working.

Bad combo obviously

**Reply by Kevin3-00** [reply]
> > First I commend your efforts in making this tool,I loaded combos of 1.5 million lines sometimes lesser but at the end no single smtp found.What could be the reason since I checked all the settings and followed your instructions.Please I will like to know if this tool is working.
> 
> Bad combo obviously

Thanks both bad combo and the server I was using to crack from.
